Scripts/Python/xIniHelper.py

```diff
@@ -153,40 +153,32 @@ def removeEntry(self,idx):
         except IndexError:
             pass
 
-    def findByCommand(self,cmd,idx=0):
-        idx = 0
-        for entry in self.entries[idx:]:
+    def findByCommand(self, cmd, idx=0):
+        for index, entry in enumerate(self.entries):
             if entry.command == cmd:
-                return entry,idx
-            idx += 1
-        return None,-1
+                return entry, index
+        return None, -1
 
-    def findByFirstValue(self,value,idx=0):
-        idx = 0
-        for entry in self.entries[idx:]:
+    def findByFirstValue(self, value, idx=0):
+        for index, entry in enumerate(self.entries):
             if len(entry.values) > 0 and entry.values[0] == value:
-                return entry,idx
-            idx += 1
-        return None,-1
+                return entry, index
+        return None, -1
 
-    def findByLastValue(self,value,idx=0):
-        idx = 0
-        for entry in self.entries[idx:]:
+    def findByLastValue(self, value, idx=0):
+        for index, entry in enumerate(self.entries):
             vlist = entry.values[-1:]
             if len(vlist) > 0:
                 if entry.values[-1:][0] == value:
-                    return entry,idx
-            idx += 1
-        return None,-1
+                    return entry, index
```

Comment:
😬 

```suggestion
            vlist = reversed(entry.values)
            if vlist and vlist[0] == value:
                return entry, index
```

Reply:
Why not just `entry and entry.values[-1]` ?

Reply:
I was being extra, I guess :trollface: 

```suggestion
            if entry.values and entry.values[-1] == value:
                return entry, index
 ```